cfy_manager/components/restservice/config/cloudify-rest.conf

```diff
@@ -26,5 +26,7 @@
     failed_logins_before_account_lock: {{ restservice.failed_logins_before_account_lock }},
     account_lock_period: {{ restservice.account_lock_period }},
     public_ip: '{{ manager.public_ip }}',
-    max_results: {{ restservice.max_results }}
+    max_results: {{ restservice.max_results }},
+    enable_tracing: {{ restservice.enable_tracing }},
+    jaeger_server_ip: '{{ restservice.jaeger_server_ip }}'
```

Comment:
better to use a general name, so if in the future we will change the framework it's still true. It's better to name by functionality